src/main/java/com/google/gwt/dom/client/InputElement.java

```diff
@@ -381,6 +384,19 @@ public final void setValue(String value) {
   }-*/;
 
   /**
+   * @Mock
+   * 
+   * Set value in native element (like Selenium 'sendKeys').
+   * The new value will be available in calls to getValue().
+   * N.B. Will not trigger ValueChange events until focus lost.
+   */
+  public final void setNewValue(String value) {
+      focus();
+      setAttribute("mock.Value.new", value);
```

Comment:
this method seems to be specific to your testing habits, I think it would be better to keep this as a static helper in your project